src/oauth2.rs

```diff
@@ -384,4 +386,24 @@ mod tests {
 
         assert_eq!(Oauth2::from_address("hello@web.de"), None);
     }
+
+    #[test]
+    fn test_dc_get_oauth2_addr() {
+        let ctx = dummy_context();
+        let addr = "dignifiedquire@gmail.com";
+        let code = "fail";
+        let res = dc_get_oauth2_addr(&ctx.ctx, addr, code);
+        // this should fail as it is an invalid password
```

Comment:
i will patiently wait till @dignifiedquire 's gmail get blocked for abuse/hijacking prevention :)